docs/release-notes/8488-tqdm.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**Fixes**
+
+-  The ``determined.launch.wrap_rank`` module, often used by custom launch layers, had a bug where
+   it was improperly buffering multiple lines separated by a carriage return, such as logs emitted
+   from the popular TQDM library. That has been fixed, so TQDM logs should pass through without
+   undue buffering.
```

Comment:
:orphan:

**Bug Fixes**

-  Fix an issue where the ``determined.launch.wrap_rank`` module, often used by custom launch layers, was improperly buffering multiple lines separated by a carriage return, such as logs emitted
   from the popular TQDM library. TQDM logs will pass through without undue buffering.